spotbugs/src/main/java/edu/umd/cs/findbugs/ba/obl/ObligationAnalysis.java

```diff
@@ -222,7 +225,7 @@ public void edgeTransfer(Edge edge, StateSet fact) throws DataflowAnalysisExcept
                     Obligation resultSet = database.getFactory().getObligationByName("java.sql.ResultSet");
                     fact.deleteObligation(resultSet, edge.getTarget().getLabel());
                     if (DEBUG_NULL_CHECK) {
-                        System.out.println("Deleting " + resultSet.toString() + " on edge from comparison "
+                        LOG.debug("Deleting " + resultSet.toString() + " on edge from comparison "
```

Comment:
This could be simplified with parameterised logging, eg:

        LOG.debug("Deleting {} on edge from comparison {}", resultSet, edge.getSource().getLastInstruction());